### PR TITLE
Collapsible Sidebar on Narrower Screens

### PR DESCRIPTION
**Pull request by erikaannesmith**
hola amigos! 🇲🇽 we are making progress!

this PR: 
- fixes the SVG sizing issue [we were noticing in my last PR](https://github.com/FormidableLabs/formidable-oss-landers/pull/21#issuecomment-741049694)
- makes the sidebar collapsible on narrower screens!

**closed:**
Screen Shot 2020-12-09 at 3 26 20 PM
**open:**
Screen Shot 2020-12-09 at 3 26 25 PM

### CODE REVIEW

File: src/styles/theme.js [hunk 1]
```diff
@@ -46,7 +46,7 @@ const layout = {
   maxWidth: "80rem",
   headerHeight: "3.6rem",
   footerHeight: "18.6rem",
-  sidebarWidth: "14.6rem",
+  sidebarWidth: "15.2rem",
```
Comment (erikaannesmith) [inline src/styles/theme.js:49]: I made this a bit wider so that the entire `renature` logo was covered when the sidebar opens :)

Comment (boygirl) [reply src/styles/theme.js:49]: I like that for this lander, bu tsince this is going to be general we can't count on it

**Comment by boygirl** [comment]
@erikaannesmith this is looking great! Can you also have the sidebar collapse when the user clicks outside of the sidebar when it's open?

File: src/styles/theme.js (hunk 1 above)
Comment (erikaannesmith) [reply src/styles/theme.js:49]: ah yes that's a good point! I'll just reset it :)

**Comment by erikaannesmith** [comment]
> @erikaannesmith this is looking great! Can you also have the sidebar collapse when the user clicks outside of the sidebar when it's open?

definitely! I'll ping you when this is ready!

**Review by boygirl — APPROVED** [review]
Looking good!

**Pull request merged**